### PR TITLE
Fixes maint pill score not being tracked

### PR DESCRIPTION
**Pull request by Time-Green**
:cl:
fix: Oops, maint pill consumption counter now works
/:cl:

In my defense, I can't test any of this without DB acces

### CODE REVIEW

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -278,7 +278,7 @@
 /obj/item/reagent_containers/pill/maintenance/on_consumption(mob/M, mob/user)
 	. = ..()
 
-	M.client?.give_award(MAINTENANCE_PILL_SCORE, M)
+	M.client?.give_award(/datum/award/score/hardcore_random, M)
```
Comment (coiax) [inline code/modules/reagents/reagent_containers/pill.dm:281]: Didn't you mean
```suggestion
	M.client?.give_award(/datum/award/score/maintenance_pill, M)
```

Comment (Time-Green) [reply code/modules/reagents/reagent_containers/pill.dm:281]: hah... hah... ugh i hate myself

**Comment by ATH1909** [comment]
headrev takes floorpill; dies the prequel
headrev takes floorpill; dies

**Comment by optimumtact** [comment]
you can run mariadb in a docker and have your own db?

**Comment by Time-Green** [comment]
> you can run mariadb in a docker and have your own db?

You’re expecting too much from me. Anyway it works now, I mixed up the path and the flag which was what broke it

**Pull request merged**